tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -41,6 +41,36 @@
 def histogram(name, data, step=None, buckets=None, description=None):
     """Write a histogram summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Logs a histogram to the current log directory for later analysis in
+    TensorBoard's 'Histograms' and 'Distributions' dashboards (one histogram
```

Comment:
slightly clearer imo for the parenthetical to say "(data written using this API will appear in both places)".

Otherwise "one histogram" makes me think "only one of them? which one?"

Reply:
Done.